<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>排序</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>排序字段</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1" colspan="2">
                <widget class="QComboBox" name="comboBox">
                 <property name="currentIndex">
                  <number>-1</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>238</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="1">
                <widget class="QRadioButton" name="radioButton">
                 <property name="text">
                  <string>升序</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QRadioButton" name="radioButton_2">
                 <property name="text">
                  <string>降序</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>筛选</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="0">
                <widget class="QRadioButton" name="radioButton_3">
                 <property name="text">
                  <string>男</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1" colspan="2">
                <widget class="QRadioButton" name="radioButton_4">
                 <property name="text">
                  <string>女</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QRadioButton" name="radioButton_5">
                 <property name="text">
                  <string>全选</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>学生信息</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QTableView" name="tableView"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>学生档案</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QTableView" name="tableView_2"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>查询结果</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QTableView" name="tableView_3"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>学号</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="StuNo">
               <property name="maxLength">
                <number>6</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="tipNo">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>姓名</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Name">
               <property name="maxLength">
                <number>5</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="tipName">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>性别</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="Gender">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>男</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>女</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>108</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>年龄</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Age">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="maxLength">
                <number>3</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="tipAge">
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>年龄不能超过3位数</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>院系</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="College"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>专业</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Major"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>班级</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Class"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>电话</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="PhoneNumber">
             <property name="maxLength">
              <number>11</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="tipPhone">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>学号</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLineEdit" name="StuNo2">
             <property name="maxLength">
              <number>6</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="tipNo2">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>姓名</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLineEdit" name="Name2">
             <property name="maxLength">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="tipName2">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>性别</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="Gender2">
           <item>
            <property name="text">
             <string>男</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>女</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>奖励</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPlainTextEdit" name="Award"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>处分</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPlainTextEdit" name="Punish"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>籍贯</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="NativePlace"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>家长信息</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QTextEdit" name="ParentsInfo"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_3">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>241</width>
              <height>447</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="1" column="0">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>姓名</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QComboBox" name="comboBox_2">
               <property name="currentIndex">
                <number>0</number>
               </property>
               <item>
                <property name="text">
                 <string>男</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>女</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>性别</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>学号</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QLineEdit" name="lineEdit_3"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>年龄</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>院系</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLineEdit" name="lineEdit_4">
               <property name="maxLength">
                <number>3</number>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>专业</string>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QLineEdit" name="lineEdit_5"/>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>班级</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>电话</string>
               </property>
              </widget>
             </item>
             <item row="7" column="2">
              <widget class="QLineEdit" name="lineEdit_7">
               <property name="maxLength">
                <number>11</number>
               </property>
              </widget>
             </item>
             <item row="6" column="2">
              <widget class="QLineEdit" name="lineEdit_6"/>
             </item>
             <item row="8" column="2">
              <widget class="QTextEdit" name="textEdit"/>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="label_26">
               <property name="text">
                <string>处分</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_25">
               <property name="text">
                <string>奖励</string>
               </property>
              </widget>
             </item>
             <item row="10" column="0">
              <widget class="QLabel" name="label_27">
               <property name="text">
                <string>籍贯</string>
               </property>
              </widget>
             </item>
             <item row="11" column="0" colspan="2">
              <widget class="QLabel" name="label_28">
               <property name="text">
                <string>家长信息</string>
               </property>
              </widget>
             </item>
             <item row="10" column="2">
              <widget class="QLineEdit" name="lineEdit_8"/>
             </item>
             <item row="9" column="2">
              <widget class="QTextEdit" name="textEdit_2"/>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="lineEdit_2">
               <property name="maxLength">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLineEdit" name="lineEdit">
               <property name="maxLength">
                <number>6</number>
               </property>
              </widget>
             </item>
             <item row="11" column="2">
              <widget class="QTextEdit" name="textEdit_3"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpenDB"/>
   <addaction name="actAdd"/>
   <addaction name="actSearch"/>
   <addaction name="actDelete"/>
   <addaction name="actShow"/>
   <addaction name="actSave"/>
   <addaction name="actCancel"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actOpenDB">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/open3.bmp</normaloff>:/new/prefix1/images/open3.bmp</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开数据库</string>
   </property>
  </action>
  <action name="actAdd">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/316.bmp</normaloff>:/new/prefix1/images/316.bmp</iconset>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
   <property name="toolTip">
    <string>添加学生信息</string>
   </property>
  </action>
  <action name="actSearch">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/300.bmp</normaloff>:/new/prefix1/images/300.bmp</iconset>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
   <property name="toolTip">
    <string>查询学生信息</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/delete1.bmp</normaloff>:/new/prefix1/images/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除学生信息</string>
   </property>
  </action>
  <action name="actSave">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/save1.bmp</normaloff>:/new/prefix1/images/save1.bmp</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
  </action>
  <action name="actCancel">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/ubdo.bmp</normaloff>:/new/prefix1/images/ubdo.bmp</iconset>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="toolTip">
    <string>取消所有修改</string>
   </property>
  </action>
  <action name="actShow">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/124.bmp</normaloff>:/new/prefix1/images/124.bmp</iconset>
   </property>
   <property name="text">
    <string>显示</string>
   </property>
   <property name="toolTip">
    <string>显示</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
